qa/vagrant/src/main/java/org/elasticsearch/packaging/util/Cleanup.java

```diff
@@ -100,19 +98,12 @@ private static void purgePackagesLinux() {
         final Shell sh = new Shell();
 
         if (isRPM()) {
+            // this may leave behind config files in /etc/elasticsearch, but a later step in this cleanup will get them
             sh.runIgnoreExitCode("rpm --quiet -e elasticsearch elasticsearch-oss");
         }
 
-        if (isYUM()) {
-            sh.runIgnoreExitCode("yum remove -y elasticsearch elasticsearch-oss");
-        }
```

Comment:
On oel 6, yum downloads a lot of stuff when this is invoked. We don't call out to yum or apt get anywhere else in the tests, and we always interact with rpm or dpkg directly, so it seemed fine to take this out